packages/core-ui/styles/core.scss

```diff
@@ -151,6 +137,115 @@ SPDX-License-Identifier: Apache-2.0 */
   }
 }
 
+.core-page-title {
+  @include page-padding;
+
+  background: var(--background-default);
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: flex-end;
+  position: sticky;
+  margin-top: 4rem;
+  margin-bottom: 0.25rem;
+  padding-top: 0.5rem;
+  padding-bottom: 0;
+  top: 0;
+  transition: padding 0.3s ease-out;
+  width: 100%;
+  z-index: 5;
+
+  @media (max-width: v.$page-width-medium-threshold) {
+    top: 4rem;
+    padding-top: 0.75rem;
+    padding-bottom: 0.5rem;
+  }
+
+  .title {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    width: 100%;
+
+    > div {
+      &:last-child {
+        padding-left: 1rem;
+        flex-grow: 1;
+      }
+    }
+  }
+
+  h1 {
+    font-size: 1.75rem;
```

Comment:
Do you think this would be a better value to have as a CSS variable? `--title-font-size`?